tests/test_monitor.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+import os
+
+from monitorframe.monitor import BaseMonitor
+from monitorframe.datamodel import BaseDataModel
+
+NEW_DATA = {
+    'a': ['A', 'B', 'C'],
+    'b': [4, 5, 6],
+    'c': [7, 8, 9],
+    'd': [10, 11, 12]
+}
+
+NOTIFICATIONS = {
+    'notification_settings': {'active': True, 'username': 'testuser', 'recipients': 'testrecipient@stsci.edu'},
+}
+
+OUTPUT_DIR = {'output': '.'}
+OUTPUT_PATH = {'output': './test_output.html'}
+NAME = {'name': 'Test Monitor Name'}
+SUBPLOTS = {'subplots': True, 'subplot_layout': (2, 1)}
+LABELS = {'labels': ['a']}
+SIMPLE_SCATTER = {'x': 'b', 'y': 'c', 'plottype': 'scatter'}
+SIMPLE_LINE = {'x': 'b', 'y': 'c', 'plottype': 'line'}
+SIMPLE_IMAGE = {'x': 'b', 'y': 'c', 'z': 'd', 'plottype': 'image'}
+
+
+@pytest.fixture
+def datamodel_test_instance():
```

Comment:
Perhaps a matter of preference here, but I usually treat test functions like any other python function in regards to docstrings.  Might be a good idea to add one here (and in the other functions/classes/methods) to describe what the function does.

Reply:
To that end, it might be also useful to include a module level docstring

Reply:
Yep, good call. I just forgot to add comments and docstrings... 😞